gr-blocks/include/gnuradio/blocks/or_blk.h

```diff
@@ -37,17 +35,18 @@ namespace gr {
      *
      * Bitwise boolean or across all input streams.
      */
-    class BLOCKS_API @NAME@ : virtual public sync_block
+    template<class T>
+    class BLOCKS_API or_blk : virtual public sync_block
     {
     public:
 
-      // gr::blocks::@NAME@::sptr
-      typedef boost::shared_ptr<@NAME@> sptr;
+      // gr::blocks::or_blk::sptr
+      typedef boost::shared_ptr<or_blk> sptr;
```

Comment:
I'll admit that I haven't tried your code changes yet, nor tried compiling this change ... but shouldn't the "or_blk" here be something more like "or_blk<T>"? -Something- needs to reflect the template class here in the changes, otherwise why bother with making it a template?

Reply:
Oh, I might have make it `shared_ptr<or_blk<T> >` good catch.